### DISCUSSION TITLE
GitTogether Medellin Event - February 20-2025

### DISCUSSION THREAD

**Discussion by vanessamarely** [gittogethers/medellin#7] [General]
Let us know how you’ve enjoyed the event so far! Feel free to drop a comment with any feedback or suggestions for improvement. Also, if there are any topics you’d like to see covered in future GitTogether events, we’d be happy to hear your ideas. Thanks for being part of our community!

**Comment by vanessamarely** [comment]
First Event 2025 https://github.com/user-attachments/assets/d210f45f-09bd-43e1-b61d-34f1824264a0

https://drive.google.com/file/d/1U_-i1xSixhSgRmxfez5icUIKRC3Tbde7/view?usp=drivesdk

**Comment by mauosp** [comment]
CI/CD of microservices in Multicloud Environments: https://github.com/mauosp/multicloud-in-git-together
DOC: [GitHubActions.pdf](https://github.com/user-attachments/files/18911594/GitHubActions.pdf)

**Comment by david4tech** [comment]
Infrastructure as a Code (IaC) in AWS with GitHub.
DOC: [IaC in AWS and GitHub.pdf](https://github.com/user-attachments/files/18913074/IaC.in.AWS.and.GitHub.pdf)
